modules/textures/src/lib/utils/slice-levels-util.js

```diff
@@ -0,0 +1,24 @@
+export function sliceLevels(data, options) {
```

Comment:
Let's have a d.ts file with a rich header explaining this function.

I prefer a more descriptive name, like `extractMipmapImages` or something like that.